dist/install

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# From https://github.com/ipfs/go-ipfs/blob/ccef991a194beaedf009b1d0702b1150db3da0a6/cmd/ipfs/dist/install.sh
+#
+# Installation script for threads. It tries to move $bin in one of the
+# directories stored in $binpaths.
+
+INSTALL_DIR="$(dirname "$0")"
+
+threadsbin="$INSTALL_DIR/threads"
+threadsdbin="$INSTALL_DIR/threadsd"
+binpaths="/usr/local/bin /usr/bin"
+
```

Comment:
I guess this isn't really going to cut it for windows as an install script. Should I just remove it from that archive?

Reply:
okay, omitting 'install' from the windows archive